AppleMusicClone/Views/MainView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MainView.swift
+//  AppleMusicClone
+//
+//  Created by Hankyu Lee on 2022/07/05.
+//
+
+import SwiftUI
+
+struct MainView: View {
+    var body: some View {
+        TabView {
+            
+            //TODO: 각 뷰 넣기
+            Text("1")
+                .tabItem {
+                    Image(systemName: "play.circle.fill")
+                    Text("지금 듣기")
+                }
+            
+            Text("2")
+                .tabItem {
+                    Image(systemName: "dot.radiowaves.left.and.right")
+                    Text("라디오")
+                }
+
+            Text("3")
+                .tabItem {
+                    Image(systemName: "play.circle.fill")
+                    Text("지금 듣기")
+                }
+
+            Text("4")
```

Comment:
```suggestion
            SearchView()
```
`SearchView()`가 아닌 `Text("4")` 로 main을 올리신 이유가 있을까요~?

Reply:
알려주셔서 감사합니다. 다소 급하게 올리느라 실험 코드가 올라간 것 같습니다..! 곧 수정하겠습니다!